packages/roosterjs-content-model/lib/modelApi/common/mergeModel.ts

```diff
@@ -259,30 +263,56 @@ function insertBlock(markerPosition: InsertPoint, block: ContentModelBlock) {
     }
 }
 
-function applyDefaultFormat(group: ContentModelBlockGroup, format: ContentModelSegmentFormat) {
+function applyDefaultFormat(
+    group: ContentModelBlockGroup,
+    format: ContentModelSegmentFormat,
+    applyDefaultFormatOption: 'MergeCurrentFormat' | 'ApplyCurrentFormat'
+) {
     group.blocks.forEach(block => {
         switch (block.blockType) {
             case 'BlockGroup':
-                applyDefaultFormat(block, format);
+                applyDefaultFormat(block, format, applyDefaultFormatOption);
                 break;
 
             case 'Table':
                 block.rows.forEach(row =>
                     row.cells.forEach(cell => {
-                        applyDefaultFormat(cell, format);
+                        applyDefaultFormat(cell, format, applyDefaultFormatOption);
                     })
                 );
                 break;
 
             case 'Paragraph':
                 block.segments.forEach(segment => {
                     if (segment.segmentType == 'General') {
-                        applyDefaultFormat(segment, format);
+                        applyDefaultFormat(segment, format, applyDefaultFormatOption);
                     }
 
-                    segment.format = { ...format, ...segment.format };
+                    segment.format =
+                        applyDefaultFormatOption == 'MergeCurrentFormat'
+                            ? { ...format, ...segment.format }
+                            : {
+                                  ...format,
+                                  ...getSemanticFormat(segment),
+                              };
                 });
                 break;
         }
     });
 }
+
+function getSemanticFormat(segment: ContentModelSegment): ContentModelSegmentFormat {
+    const result: ContentModelSegmentFormat = {};
+
+    if (segment.format.fontWeight) {
+        result.fontWeight = segment.format.fontWeight;
+    }
+    if (segment.format.italic) {
```

Comment:
Similar here, what if italic is explicitly set to false?

Reply:
Checking what Word does, if the source is not italic. We should not use Italic. 

Also if the target is italic and the source is not italic, upon merging we would set the pasted content to italic.